index.html

```diff
@@ -6688,8 +6708,12 @@ <h2 id="AudioListener">
           <dd>
             <p>
               Sets the x coordinate position of the audio listener in a 3D
-              Cartesian coordinate space. The default value is 0. This
-              parameter is a-rate.
+              Cartesian coordinate space. <a><code>PannerNode</code></a>
+              objects use this position relative to individual audio sources
```

Comment:
I don't think this should be here since we're only talking about the x coordinate. This should be moved somewhere else, like what was done for forwardXYZ for the listener.